src/utils/table-list-utils.ts

```diff
@@ -0,0 +1,32 @@
+import { SortByDirection, ThProps } from '@patternfly/react-table';
+import { HeaderFunc } from '~/shared/components/table/Table';
+
+export type ColumnConfig = {
+  title: string;
+  className: string;
+  sortable?: boolean;
```

Comment:
How about we add style now to ensure it is more flexible?

Reply:
that's a good idea, I'll update the code!